_maps/map_files/TgStation/tgstation.2.1.3.dmm

```diff
@@ -6487,8 +6487,6 @@
 "cuM" = (/turf/space,/turf/simulated/wall/shuttle{icon_state = "swall_f9"; dir = 2},/area/shuttle/pod_4)
 "cuN" = (/turf/space,/turf/simulated/wall/shuttle{dir = 2; icon_state = "swall_f10"; layer = 2},/area/shuttle/pod_4)
 "cuO" = (/turf/simulated/floor/plating,/obj/structure/shuttle/engine/propulsion/burst{dir = 4},/turf/simulated/wall/shuttle{icon_state = "swall_f6"; dir = 2},/area/shuttle/pod_4)
-"cuP" = (/obj/machinery/atmospherics/pipe/manifold/general/visible{dir = 8},/turf/simulated/mineral/bscrystal,/area/atmos)
-"cuQ" = (/obj/machinery/atmospherics/pipe/simple/green/visible,/turf/simulated/mineral/uranium,/area/atmos)

```

Comment:
sorry, giving atmos free uranium and bluespace crystals in their mixing room IS OP.